src/Core/Framework/DataAbstractionLayer/Exception/InvalidPriceFieldTypeException.php

```diff
@@ -2,22 +2,31 @@
 
 namespace Shopware\Core\Framework\DataAbstractionLayer\Exception;
 
+use Shopware\Core\Checkout\Cart\CartException;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
-use Shopware\Core\Framework\ShopwareHttpException;
+use Symfony\Component\HttpFoundation\Response;
 
+/**
+ * @deprecated tag:v6.8.0 - reason:remove-exception - Will be removed, use CartException::invalidPriceFieldTypeException() instead
+ */
 #[Package('framework')]
-class InvalidPriceFieldTypeException extends ShopwareHttpException
+class InvalidPriceFieldTypeException extends CartException
```

Comment:
if it will be removed, you should not change the parent. technically this is a break and not needed, if it got removed anyway.